README.rst

```diff
@@ -98,35 +101,31 @@ If you want to setup a development environment start like this::
     source venv/bin/activate
     pip install -r requirements.txt
     
-You may additionally want to set up your environment for testing, by adding nose and the additional dependencies::
+You may additionally want to set up your environment for testing, by adding the
+additional dependencies::
 
-    pip install nose
     pip install -r tests/requirements.txt
 
 You may also want to read the blog post about development and debugging at
 https://www.privacyidea.org/privacyidea-development-howto/
 
-Getting and updating submodules
-===============================
-
-Some authentication modules and the admin client are located in git submodules.
-To fetch the latest release of these run::
-
-   git submodule init
-   git submodule update
-
-Later you can update the submodules like this::
-
-   git pull --recurse-submodules
-
 Running it
 ==========
 
-Create the database and encryption key::
+First You need to create a `config-file <https://privacyidea.readthedocs
+.io/en/latest/installation/system/inifile.html>`_.
+
+Then create the database and encryption key::
 
     ./pi-manage createdb
     ./pi-manage create_enckey
 
+If You plan to use the database for a longer time, You should `stamp <https://privacyidea
```

Comment:
"for a longer time" sounds a bit strange to me. I know what you mean: Stamping is not necessary if I simply use the system like "now", within one release, fix a bug, do a PR and delete my whole dev folder.

I am not sure how to say this. Maybe s.th. like "If you want to have the development database to be upgradeable you need to stamp it like this::"

Reply:
Fixed in 00d3330